ugvc/mrd/featuremap_consensus_utils.py

```diff
@@ -90,23 +93,36 @@ def write_a_pileup_record(
     for field in fields_to_collect["string_list_fields"]:
         if field in record_dict:
             rec.info[field] = "|".join(record_dict[field])
+    for field in fields_to_collect["boolean_fields"]:
+        if field in record_dict:
+            rec.info[field] = "|".join(["T" if f else "F" for f in record_dict[field]])
+    for field in fields_to_collect["fields_to_write_once"]:
+        if field in record_dict:
+            if [f == record_dict[field][0] for f in record_dict[field]].count(False) == 0:
```

Comment:
1. This syntax is a bit confusing, I think it's better to use "any" or "all", e.g "if all([f == record_dict[field][0] for f in record_dict[field]])"
2. I don't understand the logic here, could you explain? You are checking that the list is just one unique value? If it's not, what happens?

Reply:
1. OK
2. if not, it doesn't write the entry
do you suggest otherwise? to raise an error?

Reply:
I think raising an error in this case is better, since you do expect them all to be the same